pkgs/tools/networking/ipgrep/default.nix

```diff
@@ -1,23 +1,25 @@
-{ stdenv, fetchFromGitHub, pythonPackages }:
+{ stdenv, fetchFromGitHub, python3Packages }:
 
-pythonPackages.buildPythonApplication rec {
-  version = "1.0";
+python3Packages.buildPythonApplication rec {
+  version = "1.0.1";
   pname = "ipgrep";
 
+  disabled = python3Packages.isPy27;
+
   src = fetchFromGitHub {
     owner = "jedisct1";
     repo = pname;
     rev = version;
-    sha256 = "1qaxvbqdalvz05aplhhrg7s4h7yx4clbfd50k46bgavhgcqqv8n3";
+    hash = "sha256:02z5b5slalza27m5gyqsksrzkv0b5284dld056xbxy0cai85rf1n";
```

Comment:
when using the `hash` attribute, the value should be of SRI format

Reply:
I'd missed that. I typically use the output of the build command:

trying https://github.com/jedisct1/ipgrep/archive/1.0.1.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   122  100   122    0     0    500      0 --:--:-- --:--:-- --:--:--   502
100  3554    0  3554    0     0   7194      0 --:--:-- --:--:-- --:--:--  7194
unpacking source archive /build/1.0.1.tar.gz
hash mismatch in fixed-output derivation '/nix/store/7011gnrnxk9zbwq0n1agnijn9jijsw4p-source':
  wanted: sha256:147h11szscb11zg0862gdpx9xr02jp1v9j02dmbwzzmgddvz8p5b
  got:    sha256:02z5b5slalza27m5gyqsksrzkv0b5284dld056xbxy0cai85rf1n

Maybe it would be an idea to print the value of the hash in sri format here too?

Reply:
that will be the case in Nix 3.0 :+1: